src/data/Repository.js

```diff
@@ -7,19 +7,18 @@ class Repository {
   /**
    * Uploads screenshots to OwlEyes and returns heatmaps showing potential defects.
    * @param {Array<File>} files
-   * @returns {Array<Promise<Image>>}
+   * @returns {Array<Promise<{ { original_img: String, res_img: String, bug_type: Array<String> } }>>}
    */
   uploadImages(images) {
     return images.map((image) => {
       return encodeFileAsBase64DataUrl(image)
         .then((encodedImage) =>
-          fetch("https://8uxam9kkod.execute-api.ap-southeast-2.amazonaws.com/Prod/owleye/", {
+          fetch("https://ekh9wzayy6.execute-api.ap-southeast-2.amazonaws.com/Prod/owleye/", {
```

Comment:
It would be best if you assign the URLs to constants.
Ideally they should be in the `.env` file.

Reply:
Moved them to `.env`